indica/bayesmodels.py

```diff
@@ -73,17 +83,25 @@ def _build_bckc(self, params: dict, **kwargs):
     def _ln_likelihood(self):
         ln_likelihood = 0
         for key in self.quant_to_optimise:
-            # TODO: What to use as error?  Assume percentage error if none given...
-            # Float128 is used since rounding of small numbers causes
-            # problems when initial results are bad fits
-            model_data = self.bckc[key].values.astype("float128")
+            # Float128 since rounding of small numbers causes problems
+            # when initial results are bad fits
+            model_data = self.bckc[key].astype("float128")
             exp_data = (
-                self.data[key]
-                .sel(t=self.plasma.time_to_calculate)
-                .values.astype("float128")
+                self.data[key].sel(t=self.plasma.time_to_calculate).astype("float128")
             )
-            _ln_likelihood = np.log(gaussian(model_data, exp_data, exp_data * 0.10))
-            ln_likelihood += np.nanmean(_ln_likelihood)
+
+            exp_error = exp_data * 0.10  # Assume percentage error if none given.
```

Comment:
Suggestion:
I would move the percentage_error value (0.1 presently) to the class init and make it a class attribute. This way you can decide.